lib/services/azuresqldbfailovergroup/cmd-update.js

```diff
@@ -0,0 +1,52 @@
+/* jshint camelcase: false */
+
+var _ = require('underscore');
+var HttpStatus = require('http-status-codes');
+
+var common = require('../../common');
+
+var sqldbfgUpdate = function (params) {
+
+    this.update = function (next) {
+        // The updated instance will be stored in the broker database.
+        var updatedInstance = _.extend({}, params.instance);
+        updatedInstance['last_operation'] = 'update';
+
+        // Update the old userRoles and userPermissions stored in the broker DB when the server password is changed manually outside of Cloud Foundry
+        if (params.requested.parameters && (params.requested.parameters.userRoles || params.requested.parameters.userPermissions)) {
+            var newUserRoles = params.requested.parameters.userRoles;
+            updatedInstance.parameters.userRoles = newUserRoles;
+            var newUserPermissions = params.requested.parameters.userPermissions;
+            updatedInstance.parameters.userPermissions = newUserPermissions;
+
+            // This only modifies the broker database so we can exit without the need to poll
```

Comment:
Could you please explain more on why we don't need to send request using SDK? Typically, we should send a request to Azure server to change the permission, userrole, etc...

Reply:
Oh, it doesn't change the roles or permissions for bound users. It is for users in new bindings. Making an effect on existing bindings might lead something unstable to the application.

Reply:
I will update the doc for this.

Reply:
Got that. Thanks for the explanation.